src/js/wasm_module.js

```diff
@@ -12,18 +12,15 @@
  * limitations under the License.
  */
 
+const { nodejs } = require("./util");
```

Comment:
Does this import patter work here or do we need to do the UMD approach defined here: https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm

Reply:
The module output format is CJS, so this is correct.